### DISCUSSION TITLE
How to disable only the construct for bold text but keep italics?

### DISCUSSION THREAD

**Discussion by rfgamaral** [micromark/micromark#148] [Q&A] [answered]
Hi there,

I looked through the documentation and some old discussion/issue where I found how to write a micromark extension to disable constructs with `disable`.

The `attention` construct is the one that will disable both bold and italics, but I'm looking for a way to disable only bold, while keeping italics around.

Similarly, I also want to disable ordered lists, but not unordered lists, but the `list` construct disables both.

I can't find a way to do it, and I'm wondering if it's possible at all to do what I need. Thoughts?

**Comment by wooorm** [comment, marked as answer]
Hey!

How would what you want work? What would for example `*a*, **b**, and ***c***` turn into? And can you show example input/output of lists?

Why do you want this? Why do you want to come up with your own non-standard language while still carrying all the bagage of markdown? Why don’t you go with AsciiDoc, reStructuredText, Org-mode, or HTML?

**Comment by rfgamaral** [comment]
Hey @wooorm,

Thanks for getting back to me :)

> How would what you want work?

That's a good question, which forced me to double-check our current implementation, and I realized that we don't actually have this distinction either, but from memory, I thought we did. I was wrong.

For context, we are using [marked](https://github.com/markedjs/marked) in [Typist](https://typist.doist.dev/) to convert Markdown into HTML, and we are in the process of replacing it with the unified ecosystem. The goal is to refactor without breaking any existing functionality, and I'm trying to replicate what we have [here](https://github.com/Doist/typist/blob/main/src/serializers/html/extensions/disabled.ts) with marked.

But again, this discussion is now moot because disabling `attention` is what we need to replicate the current behaviour we have.

> Why do you want this? Why do you want to come up with your own non-standard language while still carrying all the bagage of markdown?

Typist is a rich-text editor powered by [Tiptap](https://tiptap.dev/), and Tiptap allows you to disable pretty much any construct individually, so our serializers (Markdown to HTML and HTML to Markdown) should, ideally, support the same functionality in case someone configures the editor without support for a given construct.

We do this because we had issues in the past where certain constructs were disabled in the editor (not bold or italic, that was just a basic example for my question that turned out to be a bad example), but not on the serializer, and some content was being eaten out.

---

Anyway, I figured out everything I wanted to do, and the refactor didn't break any functionality. So it's all good. Thank you for your time.

**Reply by wooorm** [reply]
> double-check our current implementation

Which is that? marked?

> The goal is to refactor without breaking any existing functionality

Why would you spend so much time while doing exactly the same? I’d imagine you’d want some differences.

> someone configures the editor without support for a given construct.

Why would a human, who knows how the web works (you say artisans, so sounds like smart folks) *not* want bold, *not* want `ol`s? That sounds to me a bit weird for an export to do.

----

But if this is all working: perfect, then this can be finished!
